07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  font-family:arial;
+}
+
+nav {
+  background-color:black;
+  color:white;
+  width:100%;
+  height:90px;
+  margin-bottom:15px;
+  word-spacing:15px;
+  padding-left:140px;
```

Comment:
in order to get your navigation links to vertical center in your nav, add: vertical-align: middle; to 'nav'